payment_redsys/__manifest__.py

```diff
@@ -2,12 +2,16 @@
 
 {
     'name': 'Redsys Payment Acquirer',
-    'category': 'Hidden',
+    'category': 'Payment Acquirer',
     'summary': 'Payment Acquirer: Redsys Implementation',
-    'version': '9.0.1.0.0',
+    'description': 'A payment gateway to accept online payments via credit '
+                   'cards',
+    'version': '10.0.1.0.0',
     'author': "Tecnativa,"
               "Odoo Community Association (OCA)",
-    'depends': ['payment'],
+    'depends': [
+        'website_portal_sale',
```

Comment:
Bueno, Sergio, puede haber mil casos de uso. Si se puede de forma más o menos sencilla, es preferible mantener las dependencias al mínimo. Por ejemplo, lo del campo se puede ver fácilmente si existe o no (`if 'sale_order_id' in self.env['payment.transaction']._fields`).